src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 1;
+
+  return function(a) {
+    return function(b) {
+      count++;
+
+      if (count > 3 && count % 2 === 1) {
+        return 'Bzzz... Error!';
+      }
+
+      return a + b;
```

Comment:
You can use here ternary operator, it will be a much shorter
return <condition> ? 'Bzz...' : 'a + b'